tests/sanity/refresh_ignore_files

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
+target_dir = Path('.')
+
+ignore_dir = target_dir / "tests" / "sanity"
+module_dir = target_dir / "plugins" / "modules"
+module_utils_dir = target_dir / "plugins" / "module_utilss"
+ignore_dir.mkdir(parents=True, exist_ok=True)
+
+for version in ["2.9", "2.10", "2.11", "2.12"]:
+    ignore_file = ignore_dir / f"ignore-{version}.txt"
+    ignore_content = ignore_file.read_text().split("\n")
+    ignore_content.append(f"tests/sanity/refresh_ignore_files shebang!skip")
+    skip_list = [
+        "compile-2.6!skip",  # Py3.8+
+        "compile-2.7!skip",  # Py3.8+
+        "compile-3.5!skip",  # Py3.8+
+        "compile-3.6!skip",  # Py3.8+
+        "compile-3.7!skip",  # Py3.8+
+        "import-2.6!skip",  # Py3.8+
+        "import-2.7!skip",  # Py3.8+
+        "import-3.5!skip",  # Py3.8+
+        "import-3.6!skip",  # Py3.8+
+        "import-3.7!skip",  # Py3.8+
```

Comment:
Same as above, do we need this for < 3.6? I'm also a little hesitant to disable the import check on everything. We only have a few files where this is needed. I feel like a better solution would be to manage a list of files in this script we want to skip import checks on and add the skip based on that for all ansible versions.